backend/app/models/image.py

```
@@ -0,0 +1,40 @@
+from sqlmodel import Field, SQLModel, Relationship
+
+# 避免循环导入
+if TYPE_CHECKING:
+    from app.models.user import User
+
+
+class Image(SQLModel, table=True):
+    __tablename__ = "images"
+
+    id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True)
+    source_url: Optional[str] = None
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)